<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chip8 Emulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <widget class="QFrame" name="Emulator">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>961</width>
         <height>501</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="RegisterWidget" native="true">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>20</y>
         <width>241</width>
         <height>811</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <widget class="QListView" name="MemoryListView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>590</y>
          <width>241</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="RegisterTableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>241</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
       <widget class="QListView" name="RegistersListView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>740</y>
          <width>241</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QListView" name="ProgramCounterListView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>540</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Keypad" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>540</y>
         <width>271</width>
         <height>271</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <widget class="QPushButton" name="H2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H6">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H8">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>140</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H9">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>140</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>9</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="HE">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>E</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="HD">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>70</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>D</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="HC">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>0</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>C</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="HB">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>210</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>B</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="HF">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>F</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="HA">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>A</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="H0">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>210</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QListView" name="SpecialRegisters">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>780</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
